src/mod2c_core/parsact.c

```diff
@@ -958,7 +958,7 @@ s->name, s->name, s->name, s->name);
 	ITERATE(q, table) {
 		s = SYM(q);
 		if (s->subtype & ARRAY) {
-			Sprintf(buf, "static double *_t_%s[%d] = nullptr;\n",
+			Sprintf(buf, "static double *_t_%s[%d] = {nullptr};\n",
```

Comment:
```suggestion
			Sprintf(buf, "static double *_t_%s[%d]{};\n",
```
seems slightly neater to me (I realise this is my fault 🙈).

I believe
```c++
double foo[2]{7.0};
```
is not equivalent to
```c++
double foo[2]{7.0, 7.0};
```
but rather
```c++
double foo[2]{7.0, {}}; // or {7.0, 0.0}
```
so my suggestion is more explicit that the 0th element does not have special treatment.

I guess there were no tests hitting this line before.